shell/common/shell.cc

```diff
@@ -419,6 +419,8 @@ Shell::Shell(DartVMRef vm,
       volatile_path_tracker_(std::move(volatile_path_tracker)),
       weak_factory_gpu_(nullptr),
       weak_factory_(this) {
+  FML_CHECK(!settings.enable_software_rendering || !settings.enable_impeller)
```

Comment:
nits: 

Would `settings.enable_software_rendering ^ settings.enable_impeller` be more readable?